rock-paper-scissors/script.js

```
@@ -0,0 +1,38 @@
+    (playerChoice === 'rock' && computerChoice === 'scissors')||
+    (playerChoice === 'paper' && computerChoice === 'rock')||
+    (playerChoice === 'scissors' && computerChoice === 'paper')
+  ) {
+    result = "You win! " + playerChoice + " beats " + computerChoice + ".";
+  } else {
+    result = "You lose! " + computerChoice + " beats " + playerChoice + ".";
+  }
+
+  document.getElementById('result').textContent = result;
+}
+function play(playerChoice) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'play'. Consider to delete it or rename it.

'play' is defined here:


(lint/suspicious/noRedeclare)